src/tests/functional/inference_engine/core_threading.cpp

```diff
@@ -131,22 +137,24 @@ TEST_F(CoreThreadingTests, RegisterPlugins) {
 
 // tested function: GetAvailableDevices, UnregisterPlugin
 // TODO: some initialization (e.g. thread/dlopen) sporadically fails during such stress-test scenario
-TEST_F(CoreThreadingTests, DISABLED_GetAvailableDevices) {
-    InferenceEngine::Core ie;
-    runParallel([&] () {
-        std::vector<std::string> devices = ie.GetAvailableDevices();
-
-        // unregister all the devices
-        for (auto && deviceName : devices) {
-            try {
-                ie.UnregisterPlugin(deviceName);
-            } catch (const InferenceEngine::Exception & ex) {
-                // if several threads unload plugin at once, the first thread does this
-                // while all others will throw an exception that plugin is not registered
-                ASSERT_STR_CONTAINS(ex.what(), "name is not registered in the");
+TEST_F(CoreThreadingTests, GetAvailableDevices) {
```

Comment:
can we better mark test as `_DISABLED` ? Otherwise, it will pass, but does not test anything. 
Secondly, let's give better name for macro something like OV_TEST_GLIBC_VERSION_GREATER_2_34

Reply:
updated, thanks